packages/experimental-nextjs-app-support/README.md

```diff
@@ -154,6 +158,60 @@ export default function RootLayout({
 
 If you want to make the most of the streaming SSR features offered by React & the Next.js App Router, consider using the [`useSuspenseQuery`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usesuspensequery_experimental) and [`useFragment`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usefragment_experimental) hooks.
 
+### Preloading data in RSC for usage in Client Components
+
+You can also preload data in RSC to populate the cache of your Client Components.
+
+For that, follow the setup steps for both RSC and Client Components as laid out in the last two paragraphs.
+
+Then you can use the `PreloadQuery` component in your React Server Components:
+
+```jsx
+<PreloadQuery
+  query={QUERY}
+  variables={{
+    foo: 1,
+  }}
+>
+  <Suspense fallback={<>loading</>}>
+    <ClientChild />
+  </Suspense>
+</PreloadQuery>
+```
+
+> The `Suspense` boundary here is optional.
+
+This example will fetch your query in RSC, and then transport the data into your Client Component cache.
+Before the child `ClientComponent` in the example renders, a "simulated network request" for this query is started in your Client Components.
```

Comment:
```suggestion
Before the child `ClientChild` in the example renders, a "simulated network request" for this query is started in your Client Components.
```

Did you mean this component? I don't see one called `ClientComponent` in an example anywhere so just making sure this is a typo and I didn't miss something obvious.

Reply:
Yup, thanks!